llvm/lib/Transforms/FuzzIntrospector/FuzzIntrospector.cpp

```diff
@@ -336,8 +336,12 @@ std::string FuzzIntrospector::getNextLogFile() {
   std::string TargetLogName;
   std::string RandomStr = GenRandom(10);
   int Idx = 0;
+  std::string prefix = "";
+  if (getenv("SRC")) {
```

Comment:
Should we make the environment variable more explicit to fuzz introspector, i.e. not `SRC` which is an OSS-Fuzz name? I added two environment variables recently: https://github.com/ossf/fuzz-introspector/blob/50d02ba2094524bcda679605c541b2a456d039bf/llvm/lib/Transforms/FuzzIntrospector/FuzzIntrospector.cpp#L251 and https://github.com/ossf/fuzz-introspector/blob/50d02ba2094524bcda679605c541b2a456d039bf/llvm/lib/Transforms/FuzzIntrospector/FuzzIntrospector.cpp#L855  I think it would be nice to prefix the environment variables in the same way. I see that I mixed it up slightly (`FUZZ_INTROSPECTOR` and `FUZZINTRO_CONSTANTS`). How about prefixing all environment variables with `FUZZINTRO_`, and then perhaps changing `SRC` to `DST` (or perhaps `OUTDIR` or something else)?

Reply:
yes, it make sense to unify environment variable names.